kratos/geometries/line_2d_2.h

```diff
@@ -910,9 +910,8 @@ class Line2D2 : public Geometry<TPointType>
         ) const override
     {
         // We compute the distance, if it is not in the pane we
-        const Point point_to_project(rPoint);
-        Point point_projected;
-        const double distance = GeometricalProjectionUtilities::FastProjectOnLine2D(*this, point_to_project, point_projected);
+        CoordinatesArrayType point_projected;
+        const double distance = GeometricalProjectionUtilities::FastProjectOnLine2D(*this, rPoint, point_projected);
```

Comment:
please remove this check, see #7401

Reply:
But that should be fixed directly in #7401?